ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -186,7 +186,139 @@ presubmits:
         - name: JOB_NAME_SAFE
           value: e2e-libvirt
         - name: TEST_COMMAND
-          value: TEST_SUITE=openshift/conformance/parallel run-tests
+          value: |
```

Comment:
:scream:

Reply:
> :scream:

I know, right?  Hopefully we get the full parallel suite passing soon.